src/utils/Processing.ts

```diff
@@ -56,22 +54,22 @@ export const AUTOMATIC_DEPENDENCIES: AutomaticDependency[] = [
   {
     packageId: 'hl7.terminology.r4',
     version: 'latest',
-    fhirVersion: R4_OR_4B_REGEX
+    fhirVersions: ['R4', 'R4B']
   },
   {
     packageId: 'hl7.terminology.r5',
     version: 'latest',
-    fhirVersion: R5_OR_CURRENT_REGEX
+    fhirVersions: ['R5']
   },
   {
     packageId: 'hl7.fhir.uv.extensions.r4',
     version: 'latest',
-    fhirVersion: R4_OR_4B_REGEX
+    fhirVersions: ['R4', 'R4B']
   },
   {
     packageId: 'hl7.fhir.uv.extensions.r5',
     version: 'latest',
-    fhirVersion: R5_OR_CURRENT_REGEX
+    fhirVersions: ['R5']
```

Comment:
While this wasn't the primary purpose of this PR, I really like that the automatic dependencies are now associated with specific version constants rather than just a regex.